### ISSUE TITLE
Docker image is not working as expected

### ISSUE THREAD

**Issue by khatekar** [gchq/CyberChef-server#56]
We are trying to use docker image, followed all the steps mentioned in [https://github.com/gchq/CyberChef-server#docker](https://github.com/gchq/CyberChef-server#docker), after running the final command, it is complaining about  node and supportive dependencies. We tried to resolved a couple of but it moved forward and again showing dependencies issue (un supported node version).

Could anyone assist regarding the same.

**Comment by Abyss-W4tcher** [comment]
Try the following :

```dockerfile
FROM node:alpine3.10
LABEL author "Wes Lambert, wlambertts@gmail.com"
LABEL description="Dockerised version of Cyberchef server (https://github.com/gchq/CyberChef-server)"
LABEL copyright "Crown Copyright 2020"
LABEL license "Apache-2.0"
COPY . /CyberChef-server
WORKDIR /CyberChef-server
RUN npm cache clean --force && \
         npm install
ENTRYPOINT ["npm", "--prefix", "/CyberChef-server", "run", "prod"]
```

See : https://stackoverflow.com/questions/57534295/npm-err-tracker-idealtree-already-exists-while-creating-the-docker-image-for

**Comment by GCHQDeveloper1138** [comment]
Fixed in #59
